package.json

```diff
@@ -2,54 +2,28 @@
   "name": "cluster-iq-console",
   "version": "0.1.0",
   "private": true,
-  "dependencies": {
-    "@babel/runtime": "^7.22.11",
-    "@patternfly/react-core": "^5.0.0",
-    "@patternfly/react-icons": "^5.0.0",
-    "@patternfly/react-table": "^5.0.0",
-    "@testing-library/jest-dom": "^5.17.0",
-    "@testing-library/react": "^13.4.0",
-    "@testing-library/user-event": "^13.5.0",
-    "@types/enzyme": "^3.10.15",
-    "@types/jest": "^27.5.2",
-    "@types/node": "^16.18.46",
-    "@types/react": "^18.2.21",
-    "@types/react-dom": "^18.2.7",
-    "axios": "^1.5.0",
-    "date-fns": "^3.6.0",
-    "enzyme": "^3.11.0",
-    "http-proxy-middleware": "^3.0.0",
-    "patternfly": "^3.59.5",
-    "react": "^18.2.0",
-    "react-core": "^0.0.0",
-    "react-dom": "^18.2.0",
-    "react-router-dom": "^6.15.0",
-    "react-scripts": "5.0.1",
-    "typescript": "^4.9.5",
-    "web-vitals": "^2.1.4"
-  },
+  "type": "module",
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject"
```

Comment:
Missing "lint" and "format" scripts. Not sure if we still needing them. If not, it's ok, but we should remove the Makefile rules for those verbs too.

Reply:
Makefile updated with some additional improvements, PTAL